app/__main__.py

```diff
@@ -86,17 +85,19 @@ def _parse_args(args):
         'deploy', help='Deploy clusterrunner to master and slaves.', formatter_class=ClusterRunnerHelpFormatter)
     deploy_parser.add_argument(
         '-m', '--master', type=str,
-        help='The master host url (no port) to deploy the master on. This will be read from conf if unspecified, ' +
-             'and defaults to localhost.')
+        help=('The master host url (no port) to deploy the master on. This will be read from conf '
+              'if unspecified, and defaults to localhost.'))
```

Comment:
Plus one to removing the plus, but extra parens not needed, yeah? (Like on line 61/62 above.)

Samesies below.

Reply:
Yeah, the parens are not needed, but without them, pylint complains about the indent. Instead it wants the string on the same column as `help`.